app/assets/stylesheets/icon_customizations.scss

```diff
@@ -191,37 +192,46 @@
 .product-memory:before {
   content: "\e624";
 }
+.product-action:before {
+  content: "\e625";
+}
+.product-policy:before {
+  content: "\e626";
+}
+.product-policy_profile:before {
+  content: "\e627";
+}
 
 /* define  colors */
 .product-custom-1:before {
-  color: #009900;
+  color: #2d7623;
 }
 .product-custom-3:before {
-  color: #cccc33
+  color: #f5c12e
 }
 .product-custom-4:before {
-  color: #009900;
+  color: #2d7623;
 }
 .product-custom-8:before {
-  color: red;
+  color: #cc0000;
 }
 .product-custom-9:before {
-  color: #3333ff;
+  color: #00659c;
 }
 .product-custom-10:before {
-  color: #3333ff;
+  color: #00659c;
 }
 .product-custom-11:before {
-  color: #3333ff;
+  color: #00659c;
 }
 .product-custom-12:before {
-  color: #3333ff;
+  color: #00659c;
 }
 .product-custom-13:before {
-  color: #3333ff;
+  color: #00659c;
 }
 .product-custom-2:before {
-  color: #3333ff;
+  color: #00659c;
```

Comment:
Should this color be named like we do for `$color_success`, because I see `#00659c` repeated 6 times here.